.eslintrc

```diff
@@ -9,12 +9,32 @@
   ],
   "plugins": ["jsx-a11y"],
   "rules": {
+
     // driving me nuts and would have involved changing a zillion files for no real benefit
     "react/jsx-one-expression-per-line": "off",
     "quotes": "off",
     "semi": "off",
     "camelcase":"off",
     "no-unused-expressions": "off",
+    // enforce to have interfaces/type should start with "I".
+    // "@typescript-eslint/naming-convention": [
+    //   "error",
+    //   {
+    //     "selector": "interface",
+    //     "format": ["PascalCase"],
+    //     "custom": {
+    //       "regex": "^I[A-Z]",
+    //       "match": true
+    //     }
+    //   }
+    // ],
+    "no-loop-func":"error",
+    "indent": ["error", 4],
+    "no-tabs": 0,
+    "@typescript-eslint/no-unused-vars": "off",
+    "no-unused-vars": "warn",
+    "no-console": "error",
+    "no-debugger": 1,
     // matter of preference
     "import/prefer-default-export": "off",
```

Comment:
what is this for?

Reply:
@HaGuesto When there is only a single export from a module, prefer using default export over named export.